lib/puppet/provider/rabbitmq_erlang_cookie/ruby.rb

```diff
@@ -16,12 +16,12 @@ def exists?
   def content=(value)
     if resource[:force] == :true # Danger!
       puppet('resource', 'service', resource[:service_name], 'ensure=stopped')
-      FileUtils.rm_rf('/var/lib/rabbitmq/mnesia')
+      FileUtils.rm_rf(resource[:rabbitmq_home] + File::PATH_SEPARATOR + 'mnesia')
```

Comment:
This results in a failure in the acceptance tests:

```
Error: /Stage[main]/Rabbitmq::Config/Rabbitmq_erlang_cookie[/var/lib/rabbitmq/.erlang.cookie]/content: change from UFZMNVXTKIVWOFKSPHHB to TESTCOOKIE failed: Invalid parameter rabbitmq_home(:rabbitmq_home)
```

This is because rabbitmq_home isn't a parameter for the rabbitmq_erlang_cookie type. Same with rabbitmq_user and rabbitmq_group below.

You should be able to add them as parameters in lib/puppet/type/rabbitmq_erlang_cookie.rb.